tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -379,6 +379,9 @@ def test_run_same_prompt_multiple_times(self, setup):
             include_prompt_logits=True,
             max_tokens=self.num_tokens_generate,
         )
+        output_1 = next(output_1)
```

Comment:
are these changes for streaming scenarios? if not, why are we returning as a generator in the base case?

Reply:
same comment as above